functional/tests/sample.test.mjs

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+import { expect, jest, test, afterAll, beforeAll } from "@jest/globals";
+import * as utilities from "./utilities.mjs";
+
+jest.setTimeout(20000);
+
+beforeAll(async () => {
+  console.log(await utilities.mfState(true));
+});
+
+afterAll(async () => {
+  //Stop Mock Firebolt
+  // console.log(await utilities.mfState(false));
+  console.log(await utilities.killPort(9998));
+});
+
+test(`MF Startup/Health Check`, async () => {
+  let healthCheckResponse = await utilities.callApi("/api/v1/healthcheck");
+  expect(healthCheckResponse).toBeDefined();
+  expect(healthCheckResponse.status).toBe(200);
+});
+
+test(`OPENRPC Response for CORE SDK`, async () => {
+  const response = await utilities.fireboltCommand(
+    JSON.stringify({
+      method: "accessibility.closedCaptionsSettings",
+      params: {},
+      id: 0,
+    })
+  );
+  expect(response.includes('"enabled":true')).toEqual(true);
+});
+
+test(`run MF cli command to perform heath check`, async () => {
```

Comment:
this can be moved to health-check.test.mjs ?

Reply:
in health-check I am checking the mfos health apis with custom ports only. If I include the default port's test into that file it's creating issue while running the mfos that's why I kept it separate.